### PR TITLE
display menu and list new product in index

### PR DESCRIPTION
**Pull request by mvn-tienho-dn**
display product in index

### CODE REVIEW

File: app/Http/Frontend/routes.php [hunk 1]
```diff
@@ -12,7 +12,8 @@
     */
 
     Route::get('/', function () {
-        return view('frontend.dashboard.index');
+        $products = DB::table('products')->select('name', 'price', 'image')->skip(0)->take(6)->get();
```
Comment (ghost) [inline app/Http/Frontend/routes.php]: don't put code in routes, please move to controller

File: resources/views/frontend/layouts/nav.blade.php [hunk 2]
```diff
@@ -17,15 +17,12 @@
...
-                            <a href="#">Link 1</a>
-                        </li>
-                        <li>
-                            <a href="#">Link 2</a>
-                        </li>
-                        <li>
-                            <a href="#">Link 3</a>
-                        </li>
+                        <?php
+                            $categories = DB::table('categories')->select('name')->get();
```
Comment (ghost) [inline resources/views/frontend/layouts/nav.blade.php]: don't create query in view

**Comment by mvn-tienho-dn** [comment]
@at-vienlm please review this PR !

**Comment by mvn-tienho-dn** [comment]
@at-vienlm  please review this PR !

File: app/Models/Category.php [hunk 3]
```diff
@@ -24,4 +24,19 @@ public static function upload($photo)
...
     }
+
+    /**
+     * Show menu in navbar
+     *
+     * @return object
+     */
+    public static function getMenu()
+    {
+        try {
```
Comment (ghost) [inline app/Models/Category.php]: why you put `try catch` here

Comment (mvn-tienho-dn) [inline app/Models/Category.php]: ok, i will fix now!

**Pull request merged**